docs/superseding.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Superseding
+---
+
+In many cases, executing some tasks in the queue can render others unnecessary.
+For example, if the test tasks for a later version-control revision succeed, then tests for earlier revisions can be skipped.
+Such earlier tasks are said to be "superseded" by the later task.
+
+Taskcluster defines a convention for superseding multiple tasks, executing only one (the "primary task").
+Crucially, the primary task need not be the same task that the worker found on the queue, but may be one deeper into the queue (e.g., a change pushed to version control later).
+
+Superseding is implemented by workers, and is only supported by a few worker implementations -- consult their documentation for details.
```

Comment:
minor nit: in the taskcluster-docs site, we said _most_ workers support superseding, and here we say _only a few_ - we should probably align the two. Maybe we should say "not all" and provide a link for the curious reader that wants to know (and suggest to the user to browse worker docs to find out if a particular worker implements the feature or not).